charts/datasaur/templates/main/backend-worker-critical-deployment.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   labels:
     app: {{ .Release.Name }}-backend-critical-worker
   name: {{ .Release.Name }}-backend-critical-worker
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "5"
```

Comment:
what is the meaning of 5? 🤔

Reply:
itu priority, the less weight the more priority
aku set 5 karna mau backend/worker jalan belakangan @tonoman3g